source/reference/limitations.txt

```diff
@@ -53,6 +53,9 @@ General Limitations
 - `Queryable Execution
   <https://www.mongodb.com/docs/v6.0/core/queryable-encryption/>`__ is
   not supported.
+- If the :dbcommand:`renameCollection` or the :dbcommand:`drop` command
+  modifies a :ref:`capped collection <manual-capped-collection>`,
```

Comment:
Using "modify" to refer to "drop" seems a little misleading. What about:

"If you run the :dbcommand:`renameCollection` or the :dbcommand:`drop` command on a :ref:`capped collection <manual-capped-collection>`..."